### PR TITLE
Updated Fundamentals/Setup/Upgrading to v9

### PR DESCRIPTION
**Pull request by bergmania**
- Added run-an-unattended-upgrade
-  Version specific upgrades 8 to 9

### CODE REVIEW

File: Fundamentals/Setup/Upgrading/general-v9.md [hunk 1]
```diff
@@ -44,3 +44,56 @@ When the command completes, open the **.csproj** file to make sure the package r
...
+
+Below you will find the steps you need to take in order to upgrade your project unattended.
+
+:::tip
+Are you running a load balanced setup with multiple servers and environments?
+
+Check out the section about [Unattended upgrades in a load balanced setup](#unattended-upgrades-in-a-load-balanced-setup).
+:::
+
+### Enable the feature
```
Comment (jonat123) [inline Fundamentals/Setup/Upgrading/general-v9.md:60]:
```suggestion
### Enable the Upgrade Unattended feature
```

File: Fundamentals/Setup/Upgrading/general-v9.md [hunk 2]
```diff
@@ -44,3 +44,56 @@ When the command completes, open the **.csproj** file to make sure the package r
...
+
+1. Add the `Umbraco:Cms:Unattended:UpgradeUnattended` configuration key.
+2. Set the value of the key to `true`.
+
+#### Example from an appsettings json configuration file.
```
Comment (jonat123) [inline Fundamentals/Setup/Upgrading/general-v9.md:65]:
```suggestion
#### Example from an appsettings json configuration file
```

**Comment by jonat123** [comment]
@bergmania looks good overall, I have made two small suggestions 🙂

**Comment by bergmania** [comment]
@jonat123 , Thanks.. I committed your suggestions.. You can merge when you are happy.

**Comment by jonat123** [comment]
@bergmania looks good to me, i'll get it merged in.

**Pull request merged**